docs/lago-tutorial/scenarios/jenkins/README.txt

```diff
@@ -0,0 +1,6 @@
+This environment will consist of 3 vm's that will simulate
```

Comment:
Better use Readme.rst, so it can be included from the one in the root of the tutorials